_posts/en/newsletters/2020-02-19-newsletter.md

```diff
@@ -0,0 +1,254 @@
+  learns anything about each other's network identity.  See
+  [Teinturier's documentation][rv gist] for the scheme or read about
+  previous discussion of rendez-vous routing in [Newsletter #22][news22
+  rv].
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME:bitschmidty
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #18104][] ends support for building 32-bit x86 binaries
+  for Linux as part of the Bitcoin Core release process.  The
+  corresponding 32-bit binaries for Windows were previously removed
+  several months ago (see [Newsletter #46][news46 win32]).  The 32-bit
+  Linux binaries are still built as part of Bitcoin Core's continuous
+  integration tests and users may still build them manually, but the
+  binaries are no longer being distributed by the project due to a
+  lack of use and hands-on developer testing.
+
+- [C-Lightning #3488][] Bitcoin backend generalization FIXME:adamjonas
+
+- [C-Lightning #3500][] implements a simple solution to a problem that
+  could cause channels to become stuck with neither party able to sends
+  funds to the other.  The [stuck funds problem][bolts #728] occurs when
+  a payment would cause the party who funded the channel to become
+  responsible for paying more value than their current balance.  For
+  example, Alice funds a channel and pays Bob her full available
+  balance. Alice now can't spend any more money (as expected) but Bob
+  also can't pay Alice because that would require increasing the size of
+  the commitment transaction and its corresponding fees---fees that the
+  funder (Alice) is responsible for paying.  This renders the channel
+  unusable in both directions.  C-Lightning's merge simply restricts the
+  user, when they're the funder, from spending all of their available
+  balance, providing an effective short term fix.  An alternative
+  solution is proposed in [C-Lightning #3501][], but it's waiting on the
+  outcome of further discussion between the maintainers of all LN
+  implementations.
+
+- [C-Lightning #3489][] allows multiple plugins to attach to the
+  `htlc_accepted` plugin hook, with plans to allow multiple plugin
+  attachments to other hooks in the future.  For the `htlc_accepted`
+  hook, this allows a plugin to either reject the HTLC, resolve the HTLC
+  (i.e. claim any payment by returning the preimage), or pass the HTLC
+  onto the next plugin bound to the hook.
+
+- [C-Lightning #3477][] allows plugins to register feature flags that
+  will be sent in the node's [BOLT1][] `init` message, the [BOLT7][]
+  `node_announcement` message, or the [BOLT11][] invoice's feature bits
```

Comment:
(Looks like you are doing it but just in case) perhaps link to

- https://github.com/lightningnetwork/lightning-rfc/blob/master/01-messaging.md#the-init-message
- https://github.com/lightningnetwork/lightning-rfc/blob/master/07-routing-gossip.md#the-node_announcement-message
- https://github.com/lightningnetwork/lightning-rfc/blob/master/11-payment-encoding.md#feature-bits